### ISSUE TITLE
[BUG] Fox App keeps Crashing upon opening the app.

### ISSUE THREAD

**Issue by Skyridr** [Fox2Code/FoxMagiskModuleManager#152]
**Describe the bug**
So I decided to update the app and some modules today, as it's been a while. But ever since I've done so and rebooted my device,  the Fox app keeps crashing. Using the app v0.4.5.

I don't mind using logcat, but I have no idea how I can use it to filter it.

Rebooting in safe mode and back into normal boot makes the app work normally again. So it seems to be an issue with one of the modules. The only "new" module I installed was Magic FlashTool (Terminal Emulator). 

**Expected behavior**
For the app to open without crashing ^_^

**Screenshots**
Sorry for the large image.. no idea what the format is to resize an image uploaded to github.

![image](https://user-images.githubusercontent.com/6794064/169683849-9b389564-c1ec-4be7-ae3f-917c6d597e56.png)

This is the last thing it shows every time before the app crashes. 

**Device info:**
 - Device: Oneplus 9 Pro
 - OS: Oxygen OS (Android 12)
 - App LE2123_11_C.47

**Comment by nift4** [comment]
Without logcat, we can do nothing. You can extract useful information by just including lines with the word "AndroidRuntime"

**Comment by Skyridr** [comment]
I'll attempt to reinstall Magic FlashTool to see if that was the cause.

**Comment by Skyridr** [comment]
Seems Magic FlashTool (Terminal Emulator) is the cause. What logcat do you use to capture? If I include "AndroidRuntime", I only get WebViewFactory returns. Maybe I've done it wrongly. Never used these apps before.

**Comment by nift4** [comment]
Yes that seems useful.. Please install an WebView provider lol

**Comment by Skyridr** [comment]
XD, why would it crash the entire app though? Would it be possible as well to include an exception / notification that xx is not installed, thus disabling the module? So the Fox app can still be used, rather than make most of the mobile unusable. A lot of apps were no longer properly working.

**Comment by nift4** [comment]
Didn't you say you had the same problem in Home Assistant? Your ROM/some magisk module is heavily broken -> out of scope. no Android system should ever have no WebView

**Comment by Skyridr** [comment]
I mentioned that I saw Home Assistant in the error logs, never said anything about it having the exact same error. I don't have any issues with any app, besides when I installed that specific magisk module which broke the fox app.

**Comment by nift4** [comment]
In that case are you really sure you captured an correct and up to date log?

**Comment by Skyridr** [comment]
The error regarding Process: com.fox2code.mmm is the one I captured running the logger. Couldn't find any new errors. After I booted up into safe mode and re-enabled my old modules everything was working fine again. But only once I reinstalled the Magic FlashTool (Terminal Emulator) module, the Fox app started crashing again after I rebooted the system.

**Comment by nift4** [comment]
Well if you are in the crashing state, can you reboot and take a new log again? If its still the same, but WebView Tester works, you will need to wait for Fox2Code to take a look at this issue.

**Comment by androidacybot** [comment]
I've seen this error with WebView Manager but it's definitely not expected from something that shouldn't touch WebView.

@Skyridr are you using WebView manager by any chance? Or something similar?

**Comment by Skyridr** [comment]
The magisk module webview manager? Nope. The only one I'm using is the default one by Google (Android system WebView)

**Comment by Fox2Code** [comment]
FoxMMM system that is supposed to prevent crash when system can't provide WebView failed due to a typo.
Thank for the report, the issue has been fixed in the code.
